docs/sources/sharing/_index.md

```diff
@@ -1,9 +1,20 @@
 +++
 title = "Sharing"
-weight = 4
-draft = "true"
+weight = 85
 +++
 
-# Sharing
+# Sharing dashboards and panels
 
-This section contains pages that will help you share Grafana visualizations.
\ No newline at end of file
+Grafana allows you to share dashboards and panels with other users within an organization and in certain situations, publicly on the Web. You can share using:
+- A direct link
+- A Snapshot
+- An embedded link (for panels only)
+- An export link (for dashboards only)
+
+Refer to [Share dashboard]({{< relref "share-dashboard.md" >}}) and [Share a panel]({{< relref "share-panel.md" >}}) for more information.
+
+You must have an authorized viewer permission to see an image rendered by a direct link.
+
+The same permission is also required to view embedded links unless you have anonymous access permission enabled for your Grafana instance. You can enable [anonymous access]({{< relref "../auth/_index.md" >}}) by yourself in Grafana OSS. To enable anonymous access on a Grafana Cloud instance, contact your Customer Support.
```

Comment:
According to Ximena, you can enable anonymous access by yourself. So, keeping it as is. But I will double check with her again.

Reply:
Theoretically, only server admins have access to the config.ini file. Btw, this link needs to be updated, because there is no reference to anonymous access in _index.md. The correct link is: https://grafana.com/docs/grafana/latest/auth/overview/#anonymous-authentication

So yes, if you have a local on-prem Grafana and you have access to the config file, then you can enable this "by yourself." Most larger Grafana servers with admins will move that config file somewhere that non-admins cannot get to it, though.